app/src/main/java/com/example/easysurvey/dao/IEasySurveyDAO.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.easysurvey.dao
+
+import com.example.easysurvey.dto.Survey
+import retrofit2.http.GET
+import retrofit2.Call
+
+//dont forget to add an import for it
+interface IEasySurveyDAO {
+    //dont forget json from url
+    @GET("")
+    //change list if this isn't name of it
+    fun getAllSurveys(): Call<List<Survey>>
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.